### ISSUE TITLE
i3 config text color setting not having effect (MATE 1.16)

### ISSUE THREAD

**Issue by antoniotrkdz** [city41/mate-i3-applet#10]
Setting the color of the text (and possibly the border) in i3 config does not have any effect on the color of the text of the i3 workspace indicator in mate-panel.
Only the background could be changed effectively from the i3 conf. file.
I suspect the color of the text in the mate panel, set by the general theme of MATE itself, is taking precedence over the text setting in the i3 config file.
Confirmation of this could be that regardless of the color set in i3 conf., with a light theme the text of the workspace indicators are always black, as opposite to dark themes, where they are always white.

It would be nice to be able to manipulate such colors from the i3 conf. file as originally designed (and working for the background value) in order to obtain more combinations such as dark text on light background on a dark MATE theme and viceversa.

Moreover changes don't take effect immediately on reloading of i3 conf or restarting of i3, but it takes to logout and back in from mate session for the changes to be visible, which could be annoying some times.

Thanks!

**Comment by city41** [comment]
I agree with all of this. But I just don't have time to add new features I'm afraid. Pull requests are welcome.

**Comment by city41** [comment]
This issue has mostly been addressed with @tott's PR. Foreground color customizations are now honored.

**Comment by antoniotrkdz** [comment]
This is great!
I am testing it...
Thank you @tott for your work
